appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java

```diff
@@ -195,7 +208,7 @@
    * <li> If a {@link android.view.MotionEvent#ACTION_DOWN} is followed by an
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
I think appinventor uses a different naming convention pertaining to methods concerning properties. The method name hence must start with an uppercase letter. The method name is reflected in the designer and should be the name of the property.